addons/point_of_sale/migrations/9.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openupgradelib import openupgrade
+from openerp.modules.registry import RegistryManager
+
+column_defaults = {
+    'pos.config': [
+        ('barcode_nomenclature_id', None)
+    ],
+    'pos.order': [
+        ('session_id', None)
+    ],
+}
+
+
+def update_barcodes_nomenclatures(env):
+    # Create new barcode_nomenclature for each pos_config and create rules
+    # for old fields
+    cr = env.cr
+    pos_configs = env['pos.config'].search([])
+    for pos_config in pos_configs:
```

Comment:
all of this could be a lot more efficient with a `select...into` statement, but given we don't expect a lot of pos configs, we don't really care I guess